src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```
@@ -154,7 +155,8 @@ public function load(array $configs, ContainerBuilder $container): void
                 ));
         }
 
-        $container->setParameter('security.authentication.hide_user_not_found', $config['hide_user_not_found']);
+        $container->setParameter('security.authentication.hide_user_not_found', ExposeSecurityLevel::None !== $config['expose_security_errors']);
```

This is wrong. The level that is not hiding errors is `All`, not `None`